src/common/OperationsBase.ts

```diff
@@ -118,4 +124,21 @@ export class OperationsBase {
   protected topIsValid(top: number | undefined): boolean {
     return top !== undefined ? (top > 0 && top <= 1000) : true;
   }
+
+  /**
+   * Validates metadata entries.
+   * @param entries
+   */
+  protected validateMetadata(entries: GroupMetadata[]): void {
```

Comment:
No reason for this to be in the OperationsBase. Move to GroupsClient.

Reply:
Moved.